joplin/api/schema.py

```diff
@@ -67,7 +68,14 @@ def expand_dict_values(item):
     """
     dict comprehension that expands db html on each item in a dict
     """
-    return {key: try_expand_db_html(value) for (key, value) in item.items()}
+    if isinstance(item, int):
+        return item
+    try:
+        dict_values = {key: try_expand_db_html(value) for (key, value) in item.items() if item is not isinstance(item, int)}
+        return dict_values
+    except Exception as e:
+        import pdb
```

Comment:
do we want this outside of local dev?

Reply:
good point, it shouldn't hit that point anymore but I just replaced it with a log + traceback